sass/app.scss

```diff
@@ -1,6 +1,40 @@
+@import "bourbon";
+@import "neat";
+@import "normalize";
+$fa-font-path: "//netdna.bootstrapcdn.com/font-awesome/4.3.0/fonts";
+@import "font-awesome";
+
+$brand_red: rgb(162, 40, 45);
+$brand_grey: rgb(84, 83, 84);
+$border_white: rgb(245, 248, 249);
+$border_style: 8px $border_white solid;
+$drop_shadow: 0 6px 0 0 rgba(192, 195, 196, 0.7);
+
 html {
   box-sizing: border-box;
 }
+
 *, *:before, *:after {
   box-sizing: inherit;
 }
+
+body {
+  background: url(img/bg.jpg)
+}
+
+img {
+  vertical-align: top;
+}
+
+.container {
+  width: 1000px;
```

Comment:
Since your container is set to a hard `width` this will make the tablet and phone sizes scroll sideways. Change this to `max-width` and this should get things started a bit better.